### ISSUE TITLE
Crash with opengl-cb on X11

### ISSUE THREAD

**Issue by gipawu** [gnome-mpv/gnome-mpv#320]
Overview Description:
Crash sometimes when switching file on playlist with vo=opengl-cb in Xorg session

Steps to Reproduce:
1. Switch file in playlist

Actual Results:
Crash (not always but often)
In command line this error:
`[xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
gnome-mpv: xcb_io.c:165: dequeue_pending_request: Assertion `!xcb_xlib_unknown_req_in_deq' failed.`

Expected Results:
Obviously not a crash

Version:
0.13 flatpak version

Additional Information:
Nvidia 387.12

**Comment by gnome-mpv** [comment]
I can't reproduce this with Intel iGPU, so it's probably something Nvidia-specific.

**Comment by TingPing** [comment]
Also can't repro with Nvidia 387.12 in Flatpak.

**Comment by gipawu** [comment]
A little update: it seems to happen only with vo=opengl-cb and hwdec=vdpau.

**Comment by TingPing** [comment]
> A little update: it seems to happen only with vo=opengl-cb and hwdec=vdpau.

That was my exact config, the vdpau option does nothing since it tries and fails to load.

**Comment by gipawu** [comment]
You are right, I managed to get vdpau working by copying libvdpau_nvidia.so.387.12 file from nvidia runtime to libvdpau_nvidia.so in gnome platform runtime lib directory (I know, it's a ugly workaround but it works for now)
